js/README.md

```diff
@@ -114,7 +114,8 @@ $ make ios.app_deps
 
 <details><summary>Android dev requirements</summary>
 
-- An Android app **development environment**, e.g. [Android Studio](https://developer.android.com/studio/install)
+- An Android app **development environment**, e.g. [Android Studio](https://developer.android.com/studio/install) (Tested with Chipmunk 2021.2.1 .)
+  - **IMPORTANT NOTE**: There is a known issue to [Support the latest Android Studio (Giraffe)](https://github.com/berty/berty/issues/4753). To solve this we will need to update to the latest React Native version. This could be a lengthy operation and until then, although undesirable, you can mitigate this bug by using Android Studio Chipmunk.
```

Comment:
At the time we created the issue, Giraffe was the latest version. Now the latest is Jellyfish. I suggest changing to say Jellyfish.